src/main/io/displayport_msp_bf_compat.c

```diff
@@ -210,21 +209,21 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_TOTAL:
         return BF_SYM_TOTAL_DISTANCE;
-/*
-    case SYM_ALT_KM:
-        return BF_SYM_ALT_KM;
 
-    case SYM_ALT_KFT:
-        return BF_SYM_ALT_KFT;
+    case SYM_ALT_KM:
+        return 'K';
```

Comment:
This kind of change is ambiguous and confusing.

Using anything other than ? gives people the impression it is working, while it is ambiguous and confusing.